portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -801,13 +796,22 @@ else if ("always".equals(showHelpGlobal))
 				
 		boolean showPresence;
 		String globalShowPresence = ServerConfigurationService.getString("display.users.present","true");
-				
+		boolean hidePresenceTool = false;
+
+		try {
+			hidePresenceTool = site.getProperties().getBooleanProperty("presenceTool");
```

Comment:
I'd be incline to keep the property the same as the global one, rather than have a different property on the site.